client/src/leap/soledad/client/api.py

```diff
@@ -655,7 +676,24 @@ def sync(self, defer_decryption=True):
             creds=self._creds,
             defer_decryption=defer_decryption)
 
+        print "SELF SYNCER", self._dbsyncer
+
         def _sync_callback(local_gen):
+            received = self._dbsyncer.received_docs
+            self._last_received_docs = received
```

Comment:
Also, if this will ever be used outside the scope, we have to think about the variable updating period, which would be equal to the sync period.